tendermint/src/lite/errors.rs

```diff
@@ -0,0 +1,72 @@
+//! All error types tied to the light client.
+
+use crate::Hash;
+use failure::*;
+use std::time::{SystemTime, SystemTimeError};
+
```

Comment:
We could add a struct wrapping the ErrorKind as described here: https://boats.gitlab.io/failure/error-errorkind.html